.github/workflows/workflow-run-replay-verify.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
     runs-on: ${{ inputs.RUNS_ON }}
     strategy:
         matrix:
-          number: [0, 1, 2, 3, 4, 5, 6, 7] # runner number
+          number: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15] # runner number
```

Comment:
so this can't be removed / derived?

Reply:
Cannot, it is used by GitHub to spawn runners